forc/src/utils/defaults.rs

```diff
@@ -100,13 +100,22 @@ abigen!(MyContract, "out/debug/"#,
 
 async fn get_contract_instance() -> (MyContract, ContractId) {
     // Launch a local network and deploy the contract
-    let wallet = launch_provider_and_get_single_wallet().await;
+    let wallet = launch_provider_and_get_wallet().await;
 
-    let id = Contract::deploy("./out/debug/"#,
+    let id = Contract::deploy(
+        "./out/debug/"#,
         project_name,
-        r#".bin", &wallet, TxParameters::default())
-        .await
-        .unwrap();
+        r#".bin",
+        &wallet,
+        TxParameters::default(),
+        StorageConfiguration::with_storage_path(Some(
+            "./out/debug/"#,
+        project_name,
+        r#"-storage_slots.json".to_string(),
+        )),
+    )
+    .await
+    .unwrap();
```

Comment:
I tried this locally with a reasonable project name and ran `cargo fmt` and it did nothing which means that the output of this is already well formatted.